games/datingsim/master.py

```diff
@@ -105,25 +104,21 @@ def setup(self, **game_instance) -> None:
         self.game_id = self.game_instance["game_id"]
 
         self.current_turn = 0
-        self.n_turns = self.experiment['n_turns']
+        self.completed_turns = 0
+        self.max_turns = self.experiment['n_turns']
         self.num_reprompts = 0
-        self.num_completed_turns = 0
 
         self.last_response = None
         self.last_sentiment = None
 
-        self.conditions_met = False
         self.time_agreement = False
         self.location_agreement = False
         self.action_agreement = False
 
         # initialise metrics
-        self.request_counts = [0] * (self.n_turns + 1)
-        self.parsed_request_counts = [0] * (self.n_turns + 1)
-        self.violated_request_counts = [0] * (self.n_turns + 1)
-
-        # metric to save the number of turns players needed to find an agreement
-        self.turns_to_win_game = 0
+        #self.request_counts = [0] * (self.n_turns + 1)
```

Comment:
> It is one example of many where the code is commented out instead of deleting, why? Don't change it, I just want to know the logic of all those commented-instead-of-deleted places

just wanted to ask before removing them but realized we can also revert it back :D I'll ditch those lines